### PR TITLE
Fix NPE

### PR DESCRIPTION
**Pull request by baflQA**
Fixing https://github.com/allure-framework/allure-java/issues/453

### Context
Fixing https://github.com/allure-framework/allure-java/issues/453

#### Checklist
- [x] [Sign Allure CLA][cla]
- [x] Provide unit tests

[cla]: https://cla-assistant.io/accept/allure-framework/allure2

### CODE REVIEW

File: allure-httpclient/src/main/java/io/qameta/allure/httpclient/AllureHttpClientRequest.java [hunk 1]
```diff
@@ -64,7 +64,8 @@ public void process(final HttpRequest request,
         Stream.of(request.getAllHeaders())
                 .forEach(header -> builder.setHeader(header.getName(), header.getValue()));
 
-        if (request instanceof HttpEntityEnclosingRequest) {
+        if (request instanceof HttpEntityEnclosingRequest && ((HttpEntityEnclosingRequest) request)
```
Comment (pazone) [inline allure-httpclient/src/main/java/io/qameta/allure/httpclient/AllureHttpClientRequest.java:67]: Please add a unit test to cover the branch where the `request` is null

Comment (pazone) [reply allure-httpclient/src/main/java/io/qameta/allure/httpclient/AllureHttpClientRequest.java:67]: Also here is a problem with indentation

**Comment by baflQA** [comment]
Done

**Comment by baflQA** [comment]
#453

File: allure-httpclient/src/main/java/io/qameta/allure/httpclient/AllureHttpClientRequest.java [hunk 2]
```diff
@@ -15,25 +15,20 @@
...
-import org.apache.http.protocol.HttpContext;
+import static io.qameta.allure.attachment.http.HttpRequestAttachment.Builder.create;
 
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
 import java.util.stream.Stream;
 
-import static io.qameta.allure.attachment.http.HttpRequestAttachment.Builder.create;
+import io.qameta.allure.attachment.*;
```
Comment (baev) [inline allure-httpclient/src/main/java/io/qameta/allure/httpclient/AllureHttpClientRequest.java:25]: don't use wildcard imports

Comment (baflQA) [reply allure-httpclient/src/main/java/io/qameta/allure/httpclient/AllureHttpClientRequest.java]: done

**Comment by baflQA** [comment]
@baev any other remarks? Or we can merge this one?

**Comment by baev** [comment]
@baflQA @pazone thanks!
